draft-ietf-httpbis-incremental.md

```diff
@@ -89,38 +90,52 @@ downstream before receiving the complete message.
 
 {::boilerplate bcp14-tagged}
 
-The term Boolean is imported from {{!STRUCTURED-FIELDS=RFC8941}}.
-
 
 # The Incremental Header Field
 
-The Incremental HTTP header field expresses the sender's intent for HTTP
-intermediaries to start forwarding the message downstream before the entire
-message is received.
+The Incremental HTTP header field expresses the sender's preference regarding
+when HTTP intermediaries start forwarding the message downstream.
+
+This header field conveys a Structured Field Token
+({{Section 3.3.4 of STRUCTURED-FIELDS}}).
+
+This specification defines the following three Tokens:
+
+* "always": Requires incremental forwarding.
+* "preferred": Prefers incremental forwarding if feasible.
+* "not-preferred": Allow buffering before forwarding.
 
-This header field has just one valid value of type Boolean: "?1".
+Upon receiving a header section with the Incremental header field, HTTP
+intermediaries respect the sender's preference as follows:
 
-~~~
-Incremental = ?1
-~~~
+When the value is "always", intermediaries SHOULD forward the message
+incrementally with minimum buffering. If incremental delivery is impossible,
```

Comment:
Is it ever "impossible", or is it something that the server is just unwilling to support?